app/build.gradle

```diff
@@ -252,6 +258,16 @@ android {
       }
     }
   }
+  sourceSets {
+    test {
+      java.srcDirs += "$projectDir/src/testShared"
```

Comment:
Is there a reason this can't just be in the test directory?

Reply:
Yeah, androidTest and test don't have a common directory (aside from main) and I wanted to share the test functions for creating models so I didn't have to duplicate them